src/plugins/controls/public/options_list/components/options_list_popover_sorting_button.tsx

```diff
@@ -0,0 +1,161 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React, { useMemo, useState } from 'react';
+
+import {
+  EuiButtonGroupOptionProps,
+  EuiSelectableOption,
+  EuiPopoverTitle,
+  EuiButtonGroup,
+  toSentenceCase,
+  EuiButtonIcon,
+  EuiSelectable,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiToolTip,
+  EuiPopover,
+  Direction,
+} from '@elastic/eui';
+import { useReduxEmbeddableContext } from '@kbn/presentation-util-plugin/public';
+
+import { css } from '@emotion/css';
+import {
+  getCompatibleSortingTypes,
+  DEFAULT_SORT,
+  sortDirections,
+  SortBy,
+} from '../../../common/options_list/suggestions_sorting';
+import { OptionsListReduxState } from '../types';
+import { OptionsListStrings } from './options_list_strings';
+import { optionsListReducers } from '../options_list_reducers';
+
+const SORT_POPOVER_WIDTH = 220;
```

Comment:
Might be better to use a Eui size variable to get to the 220 here. That way we remain consistent with the recommendation not to have hard coded values.

Reply:
Good point 👍  Done in 93986343ab2d13ada357b60425c946d7858c60da